toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -394,3 +402,72 @@ func addCustomizerRelease(imageChroot *safechroot.Chroot, toolVersion string, bu
 
 	return nil
 }
+
+func handleSELinux(selinuxMode imagecustomizerapi.SELinux, selinuxPolicyPackage string, partitionsCustomized bool,
+	imageChroot *safechroot.Chroot,
+) error {
+	var err error
+
+	// Resolve the default SELinux mode.
+	if selinuxMode == imagecustomizerapi.SELinuxDefault {
+		selinuxMode, err = getCurrentSELinuxMode(imageChroot)
+		if err != nil {
+			return err
+		}
+	}
+
+	// If the partitions were customized, then the grub.cfg file will have been recreated from scratch and therefore
+	// the SELinux args will already be correct and don't need to be updated.
+	if !partitionsCustomized {
+		// Update the SELinux kernel command-line args.
+		err := updateSELinuxCommandLine(selinuxMode, imageChroot)
+		if err != nil {
+			return fmt.Errorf("failed to update SELinux args in grub.cfg:\n%w", err)
+		}
+	}
+
+	if selinuxMode != imagecustomizerapi.SELinuxDisabled {
+		imagerSELinuxMode, err := selinuxModeToImager(selinuxMode)
+		if err != nil {
+			return err
+		}
+
+		// Check if the selinux-policy package is installed.
+		if selinuxPolicyPackage == "" {
+			selinuxPolicyPackage = configuration.SELinuxPolicyDefault
+		}
+
+		err = imageChroot.UnsafeRun(func() error {
+			stdout, _, err := shell.Execute("rpm", "-q", selinuxPolicyPackage)
+			if err != nil {
+				// Note: RPM's stdout will say somelike like "package selinux-policy is not installed".
```

Comment:
1) if we require user to specify certain packages along with SELinux we should add it more explicitly in the docs. i do see it's part of the example but wouldn't hurt to be more clear what other things are required / conditions / recommendations with a certain property. eg im not clear if the 3 packages mentioned other than selinux-policy are requirements or recommendations.
2) is there a good reason not to install this package ourselves anyway if the user has SELinux property set but doesn't mention this package explicitly in PackagesInstall? like could it not be the default behavior?

Reply:
I've added comments to the YAML example stating what the packages are used for.

Reply:
In terms of auto-installing the package, I am mostly just following what the imager tool does. That being said, using `PackagesInstall` does give the user more flexibility in terms of which version of the package is used.